App.tsx

```diff
@@ -47,11 +47,12 @@ function CustomTabBar({ navigation }) {
 }
 
 function TabNavigator() {
+  const { state, signout } = useContext(AuthContext);
   return (
     <Tab.Navigator
       screenOptions={{ headerShown: false }}
       tabBar={(props) => <CustomTabBar {...props} />}>
-      <Tab.Screen name="CaseContactListScreen" component={CaseContactListScreen} />
+      <Tab.Screen name="CaseContactListScreen" component={CaseContactListScreen} initialParams={{staySignedIn: state.staySignedIn}} />
```

Comment:
you don't need to pass an initialParams prop
we can fetch the state using context in the component itself
see other comment